### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by wangzm509**
solve issue #3,fix some train bugs.
Now users can train models directly after cloning the code.

### CODE REVIEW

File: ACPO-model/src/models.py [hunk 1]
```diff
@@ -185,27 +184,3 @@ def get_disc(self):
         '''
         return self.disc
 
-
```
Comment (wangzhanBJCA) [inline ACPO-model/src/models.py:188]: Why delete it

File: ACPO-model/src/train.py [hunk 2]
```diff
@@ -325,14 +330,15 @@ def learn(train_data_path:str, test_data_path:str, log_dir:str, train_kwargs:dic
 
                 if args.save_model:
                     torch.save(model.state_dict(), os.path.join(log_dir, "plu.pt"))
-                    save_pb(model, log_dir, num_features)
+                    # save_pb(model, log_dir, num_features)
+                    save_pb_inline(model, log_dir, num_features)
```
Comment (wangzhanBJCA) [inline ACPO-model/src/train.py:334]: Is function name with `inline` inside reasonable?

File: ACPO-model/src/models.py (hunk 1 above)
Comment (wangzm509) [reply ACPO-model/src/models.py:188]: I will change it as a note

File: ACPO-model/src/settings.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 
 from losses import loss_dict
 
+import pdb
```
[the remark this answers is not in the document]
Comment (wangzm509) [reply ACPO-model/src/settings.py:9]: yes UR right

File: ACPO-model/src/train.py (hunk 2 above)
Comment (wangzm509) [reply ACPO-model/src/train.py:334]: i will change name soon

**Pull request closed without merging**